actors/market/tests/harness.rs

```diff
@@ -158,7 +158,7 @@ pub fn add_provider_funds(rt: &mut MockRuntime, amount: TokenAmount, addrs: &Min
     rt.set_value(amount.clone());
     rt.set_address_actor_type(addrs.provider, *MINER_ACTOR_CODE_ID);
     rt.set_caller(*ACCOUNT_ACTOR_CODE_ID, addrs.owner);
-    rt.expect_validate_caller_type((*CALLER_TYPES_SIGNABLE).clone());
+    rt.expect_validate_caller_type(vec![Type::Account, Type::Multisig]);
```

Comment:
Use CALLER_TYPES_SIGNABLE from `builtin/shared.rs` (everywhere below that uses the same two types)